openrndr-core/src/main/kotlin/org/openrndr/shape/ShapeContour.kt

```diff
@@ -375,22 +424,22 @@ data class ShapeContour(val segments: List<Segment>, val closed: Boolean, val po
     }
 
     /**
-     *
+     * Returns specified amount of points of equal distance from eachother.
      */
     fun equidistantPositions(pointCount: Int) =
             if (empty) {
                 emptyList()
             } else {
                 sampleEquidistant(
-                    adaptivePositions(),
-                    pointCount + if (closed) 1 else 0
+                        adaptivePositions(),
+                        pointCount + if (closed) 1 else 0
                 )
             }
 
     /**
      * Adaptively sample the contour into line segments while still approximating the original contour
-     * @param distanceTolerance controls the quality of the approximation
-     * @return a ShapeContour composed of linear segments
+     * @param distanceTolerance Controls the precision of the approximation
```

Comment:
"Higher values produce [ShapeContour]s with less points" (or something like that? is that actually the case?)

Reply:
Probably? I can make the change.